src/averageAges.js

```diff
@@ -37,8 +49,23 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
-}
+  let age = 0;
+
+  const f = people.filter(x => x.sex === 'f');
+
+  const m = people.map(x => (`${x.mother}`));
+
+  const res = f.filter(x => m.includes(x.name));
```

Comment:
```suggestion
  const f = people.filter(x => x.sex === 'f');

  const m = people.map(x => (`${x.mother}`));

  const res = f.filter(x => m.includes(x.name));
```
f, m, res - дуже незрозуміло